tools/earth/landcover_subindic.xml

```diff
@@ -38,7 +38,7 @@
             <when value="yes">
             </when>
             <when value="no">
-                <param name="downloaded.json" type="data" format="json" label="A file containing the land specifications" help="You can customize as needed the default file https://github.com/fair-ease/earth-critical-zone-galaxy/blob/main/landcover.json"/>
+                <param name="json" type="data" format="json" label="A file containing the land specifications" help="You can customize as needed the default file https://github.com/fair-ease/earth-critical-zone-galaxy/blob/main/landcover.json"/>
```

Comment:
is this geojson or json?

Reply:
How can I know ?

Reply:
Okay so I think it's a json

Reply:
A json with a few fixed keys, like https://github.com/galaxyproject/galaxy/blob/99ae3377f0406a3d87641d31138e749164124926/lib/galaxy/datatypes/text.py#L595

Reply:
Normally I improve the help to better explain the json and what it's useful for. And yes that's it, it's a json with some more keys for geographic info